packages/itwinui-react/src/core/ThemeProvider/ThemeProvider.tsx

```diff
@@ -164,13 +180,16 @@ export const ThemeProvider = React.forwardRef((props, forwardedRef) => {
   themeOptions.highContrast ??=
     themeProp === 'inherit' ? parent.highContrast : undefined;
 
+  future.themeBridge ??=
+    themeProp === 'inherit' ? parent.context?.future?.themeBridge : undefined;
```

Comment:
In light of the issues raised internally on Teams, we might want to always inherit this flag even when `theme` is not set `"inherit"`.

I'm not sure if it's ever useful to partially enable theme bridge for some parts of the tree, but it would still be achievable by explicitly passing `false`.

```suggestion
  future.themeBridge ??= parent.context?.future?.themeBridge;
```